frontend/package.json

```diff
@@ -86,8 +86,8 @@
     "xterm-addon-fit": "^0.5.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "if-env PUBLIC_URL react-scripts build || cross-env PUBLIC_URL=./ react-scripts build --max_old_space_size=768 && rimraf build/frontend/index.baseUrl.html",
+    "start": "npm run make-version; react-scripts start",
```

Comment:
Maybe && here would be better.

There's a pre hook feature... See here: https://docs.npmjs.com/cli/v6/using-npm/scripts

So prestart and prebuild. But I don't think this works here because it's setting environment variables?

Reply:
Cool. I will add it to pre. The `&&` I am not sure it works on Windows.

Reply:
`&&` works with cmd and powershell